### ISSUE TITLE
[ Proposal ] Porting Blanket to Android

### ISSUE THREAD

**Issue by nabilpatel4012** [rafaelmardojai/blanket#360]
Writing this to propose a request for **porting Blanket** to Android as there are some users including myself who want to use it on **Android.** 
We can go with the traditional route that is using java or kotlin for android development. I can work on java or kotlin for development. If you want to consider for both Android and iOS, then I can work on flutter as well. I have prior experience with flutter and Android in my previous Internship.
I will be waiting for your response.

**Comment by weenhall** [comment]
I have the same idea,and i need a function，Listening to the noise level when reaching the specified value,autoplay sounds

**Comment by rafaelmardojai** [comment]
I don't think this necessarily needs to be an official port of Blanket, anyone in the community can take inspitation from ir and use the sounds and icons as well. 

For example we already have this app inspired by Blanket https://f-droid.org/en/packages/ie.delilahsthings.soothingloop/, maybe some contributions are welcome there too.

**Comment by weenhall** [comment]
hi there,i just write an Android version,if there any offence,remind me,here is my project
[https://github.com/weenhall/app-noise](url)

**Comment by rafaelmardojai** [comment]
Hi @vinberg88 , sorry for the late reply, but this somehow got lost in my notifications. 

All sounds used in Blanket are sourced from third-party authors. All the info about each track licensing is here https://github.com/rafaelmardojai/blanket/blob/master/SOUNDS_LICENSING.md. All of the licenses in the list allow commercial use, they just requiere attribution so you should be fine.

**Comment by rafaelmardojai** [comment]
Since now we have various android app inspired by Blanket, I'm closing this issue.
